release/maven-central/heron-api.pom.template

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.twitter.heron</groupId>
+  <artifactId>heron-api</artifactId>
+  <version>VERSION</version>  
+  <packaging>jar</packaging>
+  <name>heron-api</name>
+  <description>Heron API</description>
+  <url>http://www.heronstreaming.io</url>
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <name>Karthik Ramasamy</name>
+      <email>kramasamy@twitter.com</email>
```

Comment:
@lewiskan - can you remove the email address? Instead can we point to the google groups?